ToDoList/ToDoList/Classes/AppDelegate.swift

```diff
@@ -9,6 +9,11 @@ import CoreData
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    var storage: StorageInterface = {
+        let storage = Storage()
+        return storage
+    }()
```

Comment:
```suggestion
    let storage = Storage()
```
I think it's not the case to use computed property here